tests/lib/rules/one-var.js

```diff
@@ -637,6 +653,22 @@ ruleTester.run("one-var", rule, {
 			options: [{ initialized: "consecutive" }],
 			languageOptions: { ecmaVersion: 2022 },
 		},
+		{
+			code: "using a = 0; using b = 1;",
+			options: ["consecutive"],
+			languageOptions: {
+				ecmaVersion: 2026,
+				sourceType: "module",
+			},
+		},
+		{
+			code: "await using a = 0; await using b = 1;",
+			options: ["consecutive"],
+			languageOptions: {
+				ecmaVersion: 2026,
+				sourceType: "module",
+			},
+		},
```

Comment:
I think these cases should be `invalid` for this rule. It might be best to remove tests for this rule and update it separately (https://github.com/eslint/eslint/issues/19792#issuecomment-3018032403).